docs/user/sbt.rst

```diff
@@ -92,13 +93,49 @@ Scala Native Version       Scala Versions
 0.4.2                      2.11.12, 2.12.{13..15}, 2.13.{4..8}
 0.4.3-RC1, 0.4.3-RC2       2.11.12, 2.12.{13..15}, 2.13.{4..8}, 3.1.0
 0.4.3                      2.11.12, 2.12.{13..15}, 2.13.{4..8}, 3.1.{0..1}
+0.4.4                      2.11.12, 2.12.{13..15}, 2.13.{4..8}, 3.1.{0..1}
+0.4.5                      2.11.12, 2.12.{13..16}, 2.13.{4..8}, 3.1.{0..3}
 ========================== ===============================================
 
-Sbt settings and tasks
-----------------------
+Sbt commands and settings
+-------------------------
+Commands
+^^^^^^^^
 
-The settings now should be set via ``nativeConfig`` in `sbt`. Setting
-the options directly is now deprecated.
+Some standard sbt commands behave differently when using the Scala Native plugin.
+
+===== ======================== ================ =========================================================
+Since Name                     Type             Description
+===== ======================== ================ =========================================================
+0.1   ``compile``              ``Analysis``     Compile Scala code to NIR
+0.1   ``run``                  ``Unit``         Compile, link and run the generated binary
+0.1   ``package``              ``File``         Similar to standard package with addition of NIR
+0.1   ``publish``              ``Unit``         Similar to standard publish with addition of NIR (1)
+0.1   ``nativeLink``           ``File``         Link NIR and generate native binary
```

Comment:
Actually, `nativeLink` is not a "standard sbt command", but is only added when ScalaNativePlugin for sbt is enabled. 
As I'm thinking about it we might extract `nativeLink` and `nativeConfig` to the list of active sbt tasks/settings. At this point we would have 3 tables: 
* sbt commands with changed behaviour
* Scala Native specific commands to interact with build tool
* Deprecated Scala Native specific commands 

in sbt we also ship two additional values: `nativeVersion` - a constant describing current version of SN that can be used in the build tool configuration and `ScalaNativeCrossVersion` - a special variant of CrossVersion dedicated for SN? proejcts. I wonder should we include them as well, probably not, since they're mostly used internally